packages/scripts/preset/config/webpack.config.prod.js

```diff
@@ -1,17 +1,16 @@
-const webpack = require('webpack');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 module.exports = () => ({
-	plugins: [
-		new webpack.optimize.UglifyJsPlugin({
-			compress: {
-				warnings: false,
-			},
-			comments: false,
-			mangle: true,
-			minimize: true,
-		}),
-		new webpack.DefinePlugin({
-			'process.env.NODE_ENV': JSON.stringify('production'),
-		}),
-	],
+	mode: 'production',
+	optimization: {
+		minimizer: [
+			new UglifyJsPlugin({
+				cache: true,
+				parallel: true,
+				sourceMap: true,
```

Comment:
Are you really sure we want sourcemap in prod mode?

Reply:
Copy/paste, Sorry I'll remove it